patches/chrome-installer-mini_installer-BUILD.gn.patch

```diff
@@ -15,8 +15,10 @@ index c365088b90b7a91faf52f5b108f64621a219b11f..b8ca81c628b565a65d973c7a05907716
 +        "$root_out_dir/brave_resources.pak",
 +        "$root_out_dir/brave_100_percent.pak",
 +        "$root_out_dir/brave_200_percent.pak",
++        "$root_gen_dir/brave/VERSION",
```

Comment:
please open an issue to fix this entire patch because we should only be changing the exe name, but I don't understand why we need the `VERSION` file as an input when the original target doesn't have the chrome `VERSION` file as an input

Reply:
This PR is not related with any executable's name changing.

This change is added because `create_installer_archive.py` used `chrome/VERSION` directly.